index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
+  var feed = (archive.content) ? archive.content : archive
   var swarm = discovery(archive, opts)
+
   archive.open(function () {
```

Comment:
Should this be feed.open? Not sure that'd work on archives if content isn't defined yet though.